source/community.applications/usr/local/emhttp/plugins/community.applications/scripts/installPluginUpdate.sh

```
@@ -1,16 +1,2 @@
-# Thin wrapper that invokes the dynamix plugin manager's `plugin update`
-# command for the plugin name passed in $1. Output and exit status come
-# directly from the underlying plugin manager script.
-#
-# Arguments:
-#   $1 - Plugin file name (e.g. `myplugin.plg`) to update.
-# Returns:
-#   Exit status of the underlying `plugin update` invocation; writes the
-#   plugin manager's progress/output to stdout.
-# ---------------------------------------------------------------------------
-/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/plugin update "$1"
+/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/plugin update $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)